app/src/main/java/com/siddharth/practiceapp/worker/MyWorker.kt

```diff
@@ -12,7 +12,10 @@ import com.siddharth.practiceapp.util.sendNotification
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.withContext
 
-// Setting up Coroutine Worker for API Call
+/**
+ * This class extends the Coroutine worker class to perform
+ * Asynchronous call to the News API using the fetchData() method
+ */
```

Comment:
Add this line : This Worker fetches top headline from the NewsApi every 15mins. 
NOTE - always be as short and clear as you can be. Don't add useless comments like extends from CoroutineWorker as it was clearly visible.

Reply:
Got it. I'll keep that in mind